recipes/sdl2_image/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "SDL2_image" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "3a3eafbceea5125c04be585373bfd8b3a18f259bd7eae3efc4e6d8e60e0d7f64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url:  https://www.libsdl.org/projects/SDL_image/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
```

Comment:
Jup sorry, I do remember your comments about this from other PRs, but I just added it back because I tried to fix the weird behavior of AppVeyor, which is giving me headaches for Python 3.4 in particular. It keeps grabbing the wrong vc versions of dependencies, specifically jpeg and libwebp. The latter however was my mistake (or actually the CI's), because it didn't build the vc10 version (which is still problematic, see https://github.com/conda-forge/libwebp-feedstock/pull/1).